packages/editor/src/components/post-format/index.js

```diff
@@ -100,8 +100,7 @@ export default function PostFormat() {
 				{ suggestion && suggestion.id !== postFormat && (
 					<p className="editor-post-format__suggestion">
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
  <tr>
    <td>
<img width="268" alt="post format before" src="https://github.com/user-attachments/assets/d451e043-e59f-41d3-bdc8-550b8cc3646c">

</td>
    <td>
<img width="275" alt="Post Format after" src="https://github.com/user-attachments/assets/674c918c-6751-457e-98ed-29e218985ba5">

</td>
  </tr>
</table>

Reply:
As @mirka mentioned in another PR, `variant="link"` `Button` components have their height set to `auto`, and therefore the `__next40pxDefaultSize` doesn't make any difference